packages/azure-http-specs/spec-summary.md

```diff
@@ -836,6 +836,218 @@ Expected response body:
 }
 ```
 
+### Azure_ResourceManager_OperationTemplates_Lro_createOrReplace
+
+- Endpoint: `put https://management.azure.com`
+
+Resource PUT operation.
+Service returns "Azure-AsyncOperation" on initial request.
+final-state-via: Azure-AsyncOperation
+
+Expected verb: PUT
+Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.OperationTemplates/orders/order1
+Expected query parameter: api-version=2023-12-01-preview
+Expected request body:
+
+```json
+{
+  "location": "eastus",
+  "properties": {
+    "productId": "product1",
+    "amount": 1
+  }
+}
+```
+
+Expected status code: 201
+Expected response header: Azure-AsyncOperation={endpoint}/subscriptions/00000000-0000-0000-0000-000000000000/providers/Azure.ResourceManager.OperationTemplates/locations/eastus/operations/lro_create_aao
+Expected response body:
+
+```json
+{
+  "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.OperationTemplates/orders/order1",
+  "name": "order1",
+  "type": "Azure.ResourceManager.Resources/orders",
+  "location": "eastus",
+  "properties": {
+    "productId": "product1",
+    "amount": 1,
+    "provisioningState": "InProgress"
+  },
+  "systemData": {
+    "createdBy": "AzureSDK",
+    "createdByType": "User",
+    "createdAt": <any date>,
+    "lastModifiedBy": "AzureSDK",
+    "lastModifiedAt": <any date>,
+    "lastModifiedByType": "User",
+  }
+}
+```
+
+Expected verb: GET
+Expected URL: {endpoint}/subscriptions/00000000-0000-0000-0000-000000000000/providers/Azure.ResourceManager.OperationTemplates/locations/eastus/operations/lro_create_aao
+
+Expected status code: 200
+Expected response body:
+
+```json
+{
+  "id": "/subscriptions/00000000-0000-0000-0000-000000000000/providers/Azure.ResourceManager.OperationTemplates/locations/eastus/operations/lro_create_aao",
+  "name": "aao",
+  "startTime": "2024-11-08T01:41:53.5508583+00:00",
+  "status": "InProgress"
```

Comment:
nit, SDK probably can only depends on the `status` (even empty body) in NRP resources

In my recent recording, I've seen 200 without body, or only
```
{
  "status" : "InProgress"
}
```
in AAO flow...

Reply:
Empty body in AAO response should mean failure right? At least Java does..
https://github.com/Azure/azure-sdk-for-java/blob/8ddf6da837a50543dfcb689887fdd6b1dcf0f8cf/sdk/core/azure-core-management/src/main/java/com/azure/core/management/implementation/polling/AzureAsyncOperationData.java#L164-L170

Reply:
OK, maybe some problem with the logging. Recording in assets appears to be correct.